assets/js/components/ExecutionResults/CheckResultDetail/CheckResultDetail.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  addPassingExpectExpectation,
+  addPassingExpectSameExpectation,
+  emptyCheckResultFactory,
+  hostFactory,
+  checksExecutionCompletedFactory,
+  checkResultFactory,
+  addCriticalExpectExpectation,
+} from '@lib/test-utils/factories';
+
+import CheckResultDetail from './CheckResultDetail';
+
+const clusterHosts = hostFactory.buildList(2);
+const [{ id: target1 }, { id: target2 }] = clusterHosts;
+const targetType = 'host';
+
+let checkResult = emptyCheckResultFactory.build({
+  targets: [target1, target2],
+});
+
+const { check_id: checkID } = checkResult;
+
+checkResult = addPassingExpectExpectation(checkResult);
+checkResult = addPassingExpectExpectation(checkResult);
+checkResult = addCriticalExpectExpectation(checkResult);
+checkResult = addCriticalExpectExpectation(checkResult);
+checkResult = addCriticalExpectExpectation(checkResult);
+checkResult = addPassingExpectSameExpectation(checkResult, 'expectation_name');
+
+const executionData = checksExecutionCompletedFactory.build({
+  check_results: [checkResultFactory.build(), checkResult],
+});
+
+export default {
+  title: 'CheckResultDetail',
+  component: CheckResultDetail,
+};
+
+export function Default() {
+  return (
+    <CheckResultDetail
+      checkID={checkID}
```

Comment:
Just a suggestion, we could use also args as a parameter for the Default function, both works :D

Reply:
You're right, having the controls available allows us to change stuff arbitrarily. Great tip

![image](https://user-images.githubusercontent.com/20770/231414587-1e428e8b-023c-4481-a930-2087ba987cc8.png)